driver-core/src/main/java/com/datastax/driver/core/ColumnMetadata.java

```diff
@@ -198,26 +198,26 @@ static Raw fromRow(Row row, VersionNumber version, ProtocolVersion protocolVersi
                 position = row.isNull(COMPONENT_INDEX) ? 0 : row.getInt(COMPONENT_INDEX);
             }
 
-            String dataTypeStr;
+            String dataType;
```

Comment:
Until now, data type resolution was done here. But now this is too early, so we only store the raw string for now, and we parse it later on.